src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_DeconstructionAssignmentOperator.cs

```diff
@@ -331,13 +345,11 @@ private ArrayBuilder<Binder.DeconstructionVariable> GetAssignmentTargetsAndSideE
                         break;
 
                     case BoundKind.TupleLiteral:
+                    case BoundKind.ConvertedTupleLiteral:
```

Comment:
>ConvertedTupleLiteral [](start = 35, length = 21)

can we get either a converted or an unconverted tuple literal here?

Reply:
We don't convert the left-hand-side of an assignment, so I think the answer is yes.

---
In reply to: [301822054](https://github.com/dotnet/roslyn/pull/37052#discussion_r301822054) [](ancestors = 301822054)